docs/DeveloperGuide.adoc

```diff
@@ -33,7 +33,7 @@
 . Observe how some tests fail. That is because they try to access the test data from the wrong directory (the working directory is expected to be the root directory, but IntelliJ runs the test with `test\` as the working directory by default). To fix this issue:
 .. Go to `Run` -> `Edit Configurations...`
 .. On the list at the left, expand `JUnit`, and remove all the test configurations (e.g. `All in test`) by selecting it and clicking on the '-' icon at the top of the list
-.. Expand `Defaults`, and ensure that `JUnit` is selected
+.. Expand `Templates` (or `Defaults` for version before 2018.2), and ensure that `JUnit` is selected
```

Comment:
I think readers may wonder which "version" we are talking about here.

Maybe we can make this clearer (though lengthier) by writing:

```
or `Defaults` if you are using an older version of IntelliJ before 2018.2
```
? What do you think?

Reply:
Sure. I think your version is clearer to the user!